rmgpy/data/kinetics/kineticsTest.py

```diff
@@ -548,15 +545,30 @@ def testfilterReactions(self):
         
         reactants = np.unique(np.array(reactants)).tolist()
         products = np.unique(np.array(products)).tolist()
+        mreactants = np.array([i.molecule[0] for i in reactants])
+        mproducts = np.array([i.molecule[0] for i in products])
+        lostReactants = mreactants[6:]
+
+        nmreactants = list(set(mreactants)-set(lostReactants))
+        nmproducts = list(set(mproducts)-set(lostReactants))
+
+        out = filterReactions(nmreactants,nmproducts,reactions)
+            
+        rset = list(set(reactions) - set(out))
+
+        msets = [set(i.reactants+i.products) for i in rset]
+        lrset = set(reactants[6:])
```

Comment:
Yes, using mreactants would require peeling the molecule object individually out of each reactant and product in each reaction so I just used reactants.